### PR TITLE
feature/TL-001: Added new Base page class

### PR DESCRIPTION
**Pull request by nuta312**
Please review my Pull request

### CODE REVIEW

File: src/main/java/pages/BasePage.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
```
Comment (nuta312) [inline src/main/java/pages/BasePage.java:6]: Please remove this line of code

**Review by kanishares — APPROVED** [review]
Approved

**Review by Ruskano — APPROVED** [review]
approve, very good

**Review by AizhanATA — APPROVED** [review]
Approved

**Review by irosh8591 — APPROVED** [review]
Approved

**Review by Bemattt — APPROVED** [review]

**Review by Gulayim2024 — APPROVED** [review]

**Comment by Ezel1711** [comment]
approved

**Pull request merged**